nouveau/README.md

```diff
@@ -0,0 +1,118 @@
+# nouveau
+
+Nouveau is a modern replacement for dreyfus/clouseau and is built on;
+
+1) the Dropwizard framework (https://dropwizard.io)
+2) Java 11+
+3) Lucene 9
+
+Nouveau transforms Apache CouchDB databases into Apache Lucene indexes at the shard level and then merges the results together.
+
+This work is currently EXPERIMENTAL and may change in ways that invalidate any existing Nouveau index.
+
+## What works?
+
+* you can define a default analyzer and different analyzers by field name.
+* sorting on text and numbers (and combinations of fields)
+* classic lucene query syntax
+* count and range facets
+* bookmark support for paginating efficiently through large results sets
+* indexes automatically deleted if database is deleted (as long as nouveau is running!)
+* integration with ken
+* integration with mango
+* integration with resharding
+* update=false
+* `_nouveau_info`
+* `_search_cleanup`
+* /openapi.{json.yaml}
+
```

Comment:
Would basic geo queries work as well as described in https://blog.cloudant.com/2022/06/28/Simple-Geospatial-Queries.html?

Reply:
definitely a (near) future thing. once we're happy with the basics we can extend functionality.